examples/ecs/run_criteria.rs

```diff
@@ -0,0 +1,47 @@
+// Simple run criteria example
```

Comment:
Can you add more color here, explaining what run criteria are and why they're useful?

Reply:
yeah sure, sorry that i forgot to